dc/core/MasterToWallChannel.h

```diff
@@ -76,19 +76,19 @@ public slots:
      * Send the given DisplayGroup to the wall processes.
      * @param displayGroup The DisplayGroup to send
      */
-    void sendAsync( DisplayGroupPtr displayGroup );
+    void sendDisplayGroup( DisplayGroupPtr displayGroup );
```

Comment:
Not absolutely necessary. It's one caveat of the new connect syntax as it cannot handle overloaded signal and slot signatures (http://wiki.qt.io/New_Signal_Slot_Syntax, http://stackoverflow.com/questions/16794695/qt5-overloaded-signals-and-slots).

I found the cast very ugly, hence the rename. But I didn't see the send(), so the documentation is at least wrong. Rename send() to sendFrame()? Is the (implementation) detail of the async/sync send relevant enough to be reflected in the function name? Or an appropriate hint the documentation would be enough?